api/database/migrations/2020_08_10_194722_transform_start_date_to_timestamp.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class TransformStartDateToTimestamp extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('events', function (Blueprint $table) {
+            $table->dropColumn('start_date');
+        });
+        Schema::table('events', function (Blueprint $table) {
+            $table->timestamp('start_date');
```

Comment:
You do not need to use a separate block for this.

Reply:
I think so, but i have an error with it.

Reply:
Got it.